src/99-appendices/03-hed.md

```diff
@@ -185,3 +185,41 @@ any present HED information will be validated using the latest version of the HE
 which is bound to result in problems.
 Hence, it is strongly RECOMMENDED that the `HEDVersion` field be included when using HED
 in a BIDS dataset.
+
+### Using HED library schemas
+
+HED also allows you to use one or more specialized vocabularies along with
+the base vocabulary. These specialized vocabularies are developed by
+communities of users and are available in the GitHub
+[hed-schema-library](https://github.com/hed-standard/hed-schema-library) repository.
+
+Example: The following `dataset_description.json` file specifies that the
+[`HED8.1.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.1.0.xml)
+base schema should be used along with the
+SCORE library for clinical neurological annotation and a test library
+located at [HED_score_0.0.1.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/score/hedxml/HED_score_0.0.1.xml) and [HED_testlib_1.0.2.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/testlib/hedxml/HED_testlib_1.0.2.xml), respectively.
+
+```JSON
+{
+  "Name": "A great experiment",
+  "BIDSVersion": "1.7.0",
+  "HEDVersion": {
+      "base": "8.1.0",
+      "libraries": {
+          "sc": "score_0.0.1",
```

Comment:
is this some "official" identity specification combining library and a version?  if not, may be worth separating out, even if more verbose:
```suggestion
          "sc": {"library": "score", "version": "0.0.1"},
```
or alike to make it explicit, and to allow in the future e.g. to expand with optional extra information (e.g. URL to the HED library which is not in **the** library of libraries i.e. [hed-schema-library](https://github.com/hed-standard/hed-schema-library)) but available from another library of libraries or may be even directly pointed to by that URL?

Reply:
Some version of what you suggest was the original plan (though URIs were never planned to be supported other than `bids:` schema paths within the dataset), but it was simplified during development after feedback from the HED community. @VisLab can provide more detail, which is escaping me.

Reply:
The suggestion is above is one that we have considered ([see hed-specification issue#156](https://github.com/hed-standard/hed-specification/issues/156)).  The actual format is only proposed at this point in the HED specification, and we would appreciate opinions and are open to changes.

We have gone through a number of versions of this, and originally we thought we would support a file within the bids data set itself and arbitrary URLs.  The HED working group consensus was to start with the standard libraries and see how it goes, since the purpose is to have standardized vocabularies to make meaningful comparisons across datasets.  

**On a related note:**
The `hed-python` tools allow a schema group to be passed into its BIDS data set constructor to override the specification in dataset_description.  Right now the `hed-javascript` public interface (which is what BIDS uses) only passes in the data set and constructs the schema group internally. 

I am mentioning this because another possibility is to allow a schema group to be passed in as part of the public interface to the `hed-javascript` validator (from bids) which would override the internal specification extracted from the `dataset_description`. This is relevant because we are going to have to do a major version bump to support the libraries because of another interface change and we could add this in as an option and allow BIDS to decide. @sappelhoff. @rwblair?

Reply:
The specification of which schemas are to be used to be used for validating a dataset are provided in a separate parameter, since `hed-javascript` cannot handle relative paths within the BIDS dataset directory and needs them parsed to use the absolute path. However, the actual `Schema` and `Schemas` types used by the validator to represent schemas are considered implementation details (despite being returned by the `validator.buildSchema` function) and not part of the stable API. In theory, the object passed as the second argument to `validateBidsDataset` can be any object conforming to the defined API, regardless of what `datasetDescription.json` says.